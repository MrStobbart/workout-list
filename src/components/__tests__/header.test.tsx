import React from "react";
import renderer from "react-test-renderer";
import Header from "../header";

describe("header component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
