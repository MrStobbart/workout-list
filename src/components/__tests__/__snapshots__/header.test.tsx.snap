// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component renders correctly 1`] = `
<div
  className="ant-page-header site-page-header ant-page-header-ghost"
  style={
    Object {
      "background": "#EFFEF3",
    }
  }
>
  <div
    className="ant-page-header-heading"
  >
    <div
      className="ant-page-header-heading-left"
    >
      <span
        className="ant-page-header-heading-title"
        title="Technical Workouts"
      >
        Technical Workouts
      </span>
      <span
        className="ant-page-header-heading-sub-title"
        title="Find your workout now!"
      >
        Find your workout now!
      </span>
    </div>
    <span
      className="ant-page-header-heading-extra"
    >
      <img
        alt="content"
        src="/logo.svg"
        style={
          Object {
            "float": "right",
            "height": "52px",
            "margin": "-10px",
            "width": "auto",
          }
        }
        width="100%"
      />
    </span>
  </div>
</div>
`;
